### PR TITLE
Support for Rails 7.1

### PR DESCRIPTION
**Pull request by dwarfmondo**
Adds support for Rails 7.1

### CODE REVIEW

**Comment by balvig** [comment]
Hello there!
Thanks for the contribution. ☀️  
I'm probably missing something, but just to clarify:

- The `Gemfile` is primarily there for development/testing purposes, and shouldn't affect whether the gem supports Rails 7.1 or not.
- The version specified is `~> 7.0` so it should [pick up any minor updates whenever running the CI](https://app.circleci.com/pipelines/github/ActiveCampaign/postmark-rails/33/workflows/3ae7c704-2f35-4582-bff1-b133e2445ae1/jobs/1032?invite=true#step-102-2736_27):
https://github.com/ActiveCampaign/postmark-rails/blob/c96a57984a9b80bda4a7603019fc2e03e0be6e8e/Gemfile#L5
- The actual `gemspec` only requires a version of `actionmailer` of `3+`: https://github.com/ActiveCampaign/postmark-rails/blob/c96a57984a9b80bda4a7603019fc2e03e0be6e8e/postmark-rails.gemspec#L22

In short: I don't think there should be a need to manually bump the version number in the `Gemspec` file, but let me know if you're having any issues!

**Comment by dwarfmondo** [comment]
@balvig You are correct, weird. I removed the reference to this branch and it works as expected. My apologies for the the false alarm. I'll go ahead and close this PR. 👍
